### ISSUE TITLE
select mps error

### ISSUE THREAD

**Issue by d1r3ct1v3** [d07RiV/diabloweb#76]
Trying to select mpq

**App version:**
DiabloWeb 1.0.33 (Retail)

**Error message:**
    
        K/@https://d07riv.github.io/diabloweb/ad404bc1f12303f5575d.worker.js?__WB_REVISION__=11b4f45812af1b8bb

And a whole bunch of similar looking ones 12 or 13 total
    

**User agent:**

    Mozilla/5.0 (Android 7.0; Tablet; rv:68.0) Gecko/68.0 Firefox/68.0

Galaxy tab s2

**Comment by d07RiV** [comment]
Which file are you selecting? Is it called DIABDAT.MPQ and if so, what is its size?

**Comment by d1r3ct1v3** [comment]
Yes that's the name and 493.53. Sorry i should have mentioned I have it working properly on my galaxy s7 edge.  But I am getting errors on both the tab s2 and lg g3.  The error on g3 is similar but not same will make a separate post for that when I get a chance...I will double check the file size.  I have tried recopying it also.  I ripped it from iso with daemon tools. Dont see why that should be an issue since it's working on my s7 edge. Thanx for response

**Comment by d07RiV** [comment]
Oh, I was hoping there'd be more information, since that doesn't really tell me where the error occurs. The red error box has the same error, then?

**Comment by d1r3ct1v3** [comment]
Not sure if I can reply here. At the end a letter b is cut off. That's all
I saw error wise pretty sure...

**Comment by d1r3ct1v3** [comment]
I figured out how to add picture to thread I put it in original post

**Comment by d07RiV** [comment]
Oh, that's odd, there are no line numbers or anything :|

**Comment by d1r3ct1v3** [comment]
And a very similar msg on my brothers lg g3.  I wonder if it has anything
to do with using daemon tools to rip file from iso, but the exact same file
works fine on my s7 edge so I dont see how that would be the case...

**Comment by AJenbo** [comment]
A bit late, but maybe still helpful. There is now a native port for Android: https://play.google.com/store/apps/details?id=org.diasurgical.devilutionx
